specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+            "$ref": "./examples/TrainBatchWithSubFolders.json"
+          }
+        }
+      },
+      "get": {
+        "summary": "List Custom Models",
+        "description": "Get information about all trained custom models",
+        "operationId": "GetCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "op",
+            "in": "query",
+            "description": "Summary or full list of models to be returned.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "full",
+              "summary"
+            ]
+          }          
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Models"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },        
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/GetModels.json"
+          }
+        }
+      }      
+    },
+    "/custom/models/{modelId}": {
+      "get": {
+        "summary": "Get Custom Model",
+        "description": "Get detailed information about a model.",
+        "operationId": "GetCustomModel",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "description": "Model identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "keys",
+            "in": "query",
+            "description": "Include list extracted keys in model information.",
+            "required": false,
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Model"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/GetModel.json"
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete Model",
+        "description": "Mark model for deletion. Model artifacts will be permanently removed within predetermined period.",
```

Comment:
Can someone "undo" delete? I'm pretty sure customer may ask for that...

Reply:
not in v2.0.